Assets/SEE/Controls/MarkAction.cs

```diff
@@ -0,0 +1,162 @@
+using System.Collections.Generic;
+using SEE.GO;
+using SEE.Net.Actions;
+using SEE.Utils.History;
+using UnityEngine;
+using SEE.Audio;
+using SEE.Game.SceneManipulation;
+using SEE.Utils;
+
+namespace SEE.Controls.Actions
+{
+    /// <summary>
+    /// Action to create a new node for a selected city.
+    /// </summary>
+    internal class MarkAction : AbstractPlayerAction
+    {
+        /// <summary>
+        /// If the user clicks with the mouse hitting a game object representing a graph node,
+        /// this graph node is a parent to which a new node is created and added as a child.

```

Comment:
This documentation is misleading. The term _node_ should refer only to graph nodes or game objects representing graph nodes. Do you mean a _marker_ instead?